downloader/src/main/kotlin/Downloader.kt

```diff
@@ -108,17 +108,21 @@ object Downloader {
         }
     }
 
+    private fun prepareOutputDirectory(outputDirectory: File) {
```

Comment:
Nit: Maybe `verifyOutputDirectory()` would be a better name? That's also what you use in the commit message, and it's IMO a bit more fitting, as we do not "prepare" the directory e.g. by copying something into it, but just verify it's empty and existing.

Reply:
Done.